README.md

```diff
@@ -1,2 +1,44 @@
-# abrt-dev-modules
-Developer modules for tracking exceptions in software
+<p align="center">
+<a href="https://retrace.fedoraproject.org/">
+    <img alt="ABRT" src="https://github.com/abrt/abrt/blob/master/icons/hicolor_apps_scalable_abrt.svg" width="123">
+</a>  
+</p>
+<h1 align=center>sABRTooth Python</h1>
+<h6 align=center>Developer module for error and exception tracking in Python applications</h6>
+
+***
+
+**sABRTooth** automatically captures errors and exceptions in your applications and collects important data for easier debugging.
+Captured errors and exceptions are reported to [ABRT Analytics](https://retrace.fedoraproject.org/faf) without any user interation.
+
+Features
+---
+**Automatic error monitoring and reporting** - Exceptions and errors in Python are automatically captured and reported.  
+
+
+Using sABRTooth
+---
+1) Install the `sABRTooth`.  
```

Comment:
The "the" is superfluous here. (Or did you mean to say "the sABRTooth package"?)